vector-test/src/test/scala/spec/geotrellis/vector/voronoi/DelaunaySpec.scala

```diff
@@ -1,114 +0,0 @@
-package geotrellis.vector.voronoi
-
-import geotrellis.vector._
-import scala.util.Random
-import scala.math.pow
-import org.apache.commons.math3.linear._
-
-import geotrellis.raster._
-import geotrellis.raster.rasterize._
-import geotrellis.raster.render._
-
-import org.scalatest.{FunSpec, Matchers}
-
-class DelaunaySpec extends FunSpec with Matchers {
```

Comment:
Should we really scrap all these tests? I think still testing our JTS backed Delaunay class makes sense.